src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -155,5 +192,20 @@ public string GenerateEndTag()
 
             return string.Format(CultureInfo.InvariantCulture, "</{0}>", TagName);
         }
+
+        /// <summary>
+        /// Changes the output of the <see cref="TagHelperOutput"/> to generate nothing.
```

Comment:
nit: remove "the output of the" since it's the `TagHelperOutput` that will generate nothing